### PR TITLE
Update exposures.md

### PR DESCRIPTION
**Pull request by BigBangData**
Warn user to use underscores in the name.
Note the effect of adding a url.

## Pre-release docs
Is this change related to an unreleased version of dbt?
- [ ] Yes: please
    - update the base branch to `next`
    - add Changelog components: `<Changelog>[New/Changed] in v0.x.0</Changelog>`
    - add links to the "New and changed documentation" section of the latest [Migration Guide](../website/docs/docs/guides/migration-guide)
- [ ] No: please ensure the base branch is `current`
- [x] Unsure: we'll let you know!

## Checklist
If you added new pages (delete if not applicable):
- [ ] The page has been added to `website/sidebars.js`
- [ ] The new page has a unique filename

If you removed existing pages (delete if not applicable):
- [ ] The page has been removed from `website/sidebars.js`
- [ ] An entry has been added to `_redirects`

### CODE REVIEW

File: website/docs/docs/building-a-dbt-project/exposures.md [hunk 1]
```diff
@@ -59,7 +59,7 @@ _Expected:_
 - **depends_on**: list of refable nodes (`ref` + `source`)
 
 _Optional:_
-- **url**
+- **url**: will enable the link to "View this exposure" in the documentation
```
Comment (mirnawong1) [inline website/docs/docs/building-a-dbt-project/exposures.md]:
```suggestion
- **url**:  enables the link to "View this exposure" in the documentation
```

File: website/docs/docs/building-a-dbt-project/exposures.md [hunk 2]
```diff
@@ -59,7 +59,7 @@ _Expected:_
 - **depends_on**: list of refable nodes (`ref` + `source`)
 
 _Optional:_
-- **url**
+- **url**:  enables the link to "View this exposure" in the documentation
```
Comment (mirnawong1) [inline website/docs/docs/building-a-dbt-project/exposures.md]:
```suggestion
- **url**:  enables the link to **View this exposure** in the dbt documentation site
```

File: website/docs/docs/building-a-dbt-project/exposures.md [hunk 3]
```diff
@@ -59,7 +59,7 @@ _Expected:_
 - **depends_on**: list of refable nodes (`ref` + `source`)
 
 _Optional:_
-- **url**
+- **url**:  enables the link to **View this exposure** in the dbt documentation site
```
Comment (mirnawong1) [inline website/docs/docs/building-a-dbt-project/exposures.md]:
```suggestion
- **url**:  enables the link to **View this exposure** in the upper right corner of the generated documentation site
```

**Comment by mirnawong1** [comment]
thanks for opening this up and for your patience @BigBangData! I'll go ahead and merge this for you :D

**Review by mirnawong1 — APPROVED** [review]

**Comment by BigBangData** [comment]
@mirnawong1 I had completely forgotten about this of course - but glad it was still useful after all this time!

**Pull request merged**